src/start/development.md

```diff
@@ -30,4 +30,18 @@ To start a new project, you can use `cargo-generate`:
 cargo generate https://github.com/aya-rs/aya-template
 ```
 
-This will prompt you for a project name. We'll be using `myapp` in this example
+This will prompt you for a project name, a type of eBPF program, and possibly for other
+choices depending on the eBPF program (for example, the 'attach direction: Ingress or Egress'
```

Comment:
You forgot to close the parentheses here. Also I'd phrase it as:

```
(for example, the attach direction for network classifier programs). 

You can also set template options directly form the command line, for example:
```

Reply:
8005ea5 - added the missing parentheses and rephrase - also avoided the repetition of "for example" in two consecutive sentences